frontend/src/lib/components/PropertyFilters/components/TaxonomicPropertyFilter.tsx

```diff
@@ -53,6 +54,7 @@ export function TaxonomicPropertyFilter({
             }}
             taxonomicGroupTypes={
                 taxonomicGroupTypes || [
+                    ...(groupTypes.length > 0 ? [...groupTypes.map((gt) => gt.group_type)] : []),
```

Comment:
Couple of issues:
1. This business logic does not belong on the component.
2. Wouldn't this enable groups everywhere groups aren't explicitly enabled? This will cause bugs since we won't support groups from the get-go everywhere.

We need a pattern to enable groups (ideally by adding one line to props in callsites) component-by-component.

Reply:
Hmm I think for point 2 it shouldn't enable groups because that's feature flagged in the `groupsModel` so when `groupTypes` returns an empty array, nothing shows up, though I can protect this more